include/nil/crypto3/marshalling/zk/types/placeholder/common_data.hpp

```diff
@@ -77,7 +77,15 @@ namespace nil {
                         nil::marshalling::types::array_list <TTypeBase,
                             nil::marshalling::types::integral<TTypeBase, octet_type>,
                             nil::marshalling::option::sequence_size_field_prefix<nil::marshalling::types::integral<TTypeBase, std::size_t>>
-                        >
+                        >,
+//                      std::size_t witness_columns;
```

Comment:
Consider having a separate mashalling structure for arithmetization parameters and use it here, because we may need to marshall this 4 numbers in other places as well.